cyences_app_for_splunk/default/data/ui/views/cs_user_inventory_table.xml

```diff
@@ -129,18 +129,18 @@
   <row>
     <panel>
       <title>User Inventory Table</title>
-      <table>
+      <table id="user_inventory_main_tbl">
         <search base="base_summary">
           <query> | search $tkn_sp_filter$
 | rename users as User, user_types as "User Types", product_names as Products
+| table user_row_selector, User, "User Types",Products, uuid
```

Comment:
add the field hear as as well is_previledged_user
* but combine with manually assigned value with one extracted from UserTypes.